src/main/kotlin/com/sparetimedevs/consonance/App.kt

```diff
@@ -62,9 +55,9 @@ fun Application.module() {
     }
 }
 
-fun main() {
-    val port = Integer.valueOf(System.getenv("PORT"))
-    Dependency.load()
+fun main(args: Array<String>) {
```

Comment:
Why do we need args? Usually, I never supply any args.

Reply:
You're right, YAGNI